src/main/resources/templates/start.html

```diff
@@ -1,77 +1,90 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org">
 <head>
-<meta charset="ISO-8859-1"/>
-<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-<meta name="viewport" content="width=device-width, initial-scale=1"/>
- <link rel="stylesheet" type="text/css" th:href="@{/webjars/bootstrap/3.3.7/css/bootstrap.min.css}"/>

```

Comment:
I don't see the `webjars` folder. I guess you copied this line from somewhere else and forgot to add the plugin 😄 so I use the CDN instead